invenio_rdm_migrator/streams/records/table_generators/versions.py

```diff
@@ -22,10 +22,10 @@ def __init__(self):
     def _generate_rows(self, parent_entry, **kwargs):
         # Version state to be populated in the end from the final state
         yield RDMVersionState(
-            latest_index=parent_entry["latest_index"],
+            latest_index=parent_entry.get("latest_index"),
```

Comment:
even if the values are None, shouldnt the key exist in the entry?

Reply:
This is because of the dict-cache I used, Since the data doesn't come back as a full row from the DB, with NULL translated to `None`.